pkg/operator/sync.go

```diff
@@ -194,10 +195,12 @@ func (optr *Operator) syncWebhookConfiguration() error {
 }
 
 func (optr *Operator) syncValidatingWebhook() error {
-	expectedGeneration := resourcemerge.ExpectedValidatingWebhooksConfiguration(mapiwebhooks.NewValidatingWebhookConfiguration().Name, optr.generations)
```

Comment:
I'm not 100% sure on the ramifications of this change here, would love some input and context. The `expectedGeneration` was removed from this methods signature per this [PR](https://github.com/openshift/library-go/pull/1256). Any potential problems that we might need to consider here?